packages/block-editor/src/hooks/dimensions.js

```diff
@@ -44,6 +44,11 @@ export const SPACING_SUPPORT_KEY = 'spacing';
 export const ALL_SIDES = [ 'top', 'right', 'bottom', 'left' ];
 export const AXIAL_SIDES = [ 'vertical', 'horizontal' ];
 
+function useVisualiserMouseOver() {
```

Comment:
```suggestion
function useVisualizerMouseOver() {
```
I think we need the US spelling here. It would also then be consistent with other references elsewhere.

Reply:
Doh! thanks - hard to get used to that `z` !